cmd/contour/cli.go

```diff
@@ -48,20 +49,16 @@ func (c *Client) dial() *grpc.ClientConn {
 		}
 		// Load the client certificates from disk
 		certificate, err := tls.LoadX509KeyPair(c.ClientCert, c.ClientKey)
-		if err != nil {
-			log.Fatalf("err: %v", err)
-		}
+		kingpin.FatalIfError(err, "err: %v", err)
 		// Create a certificate pool from the certificate authority
 		certPool := x509.NewCertPool()
 		ca, err := ioutil.ReadFile(c.CAFile)
-		if err != nil {
-			log.Fatalf("err: %v", err)
-		}
+		kingpin.FatalIfError(err, "err: %v", err)
 
 		// Append the certificates from the CA
 		if ok := certPool.AppendCertsFromPEM(ca); !ok {
 			// TODO(nyoung) OMG yuck, thanks for this, crypto/tls. Suggestions on alternates welcomed.
-			log.Fatal("failed to append ca certs")
+			kingpin.Fatalf("failed to append ca certs")
```

Comment:
I am not very sure about this. I checked kingpins' documentation and could not find `kingpin.Fatal`. Please advise.

Reply:
```suggestion
			kingpin.Fatalf("failed to append CA certs")
```

Reply:
@ShaileshSurya ^^^

Reply:
Fixed this.